spring-cloud-dataflow-docs/src/main/asciidoc/api-guide.adoc

```diff
@@ -2965,7 +2965,8 @@ The following topic provides more details:
 [[api-guide-resources-stream-logs-by-task-id]]
 ==== Get the task execution log
 
-To retrieve the logs of the task execution, query the `/tasks/logs/<ExternalTaskExecutionId>` endpoint by using the HTTP `GET` method.
+To retrieve the logs of the task execution, query the `/tasks/logs/<executionId>` endpoint by using the HTTP `GET` method.  The current default is to retrieve the logs using the external execution id (deployer id) provided by the platform.  But if you wish to retrieve the log using the execution id
```

Comment:
**[TinyNit]** I would not mention this if it was the only thing in my review as it is truly tiny nit, do you mind converting this to the good ole' [one-sentence-per-line](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line)?